### DISCUSSION TITLE
Merging multiple scoop installations

### DISCUSSION THREAD

**Discussion by geekalert** [ScoopInstaller/Scoop#5765] [Help and Questions]
Hi, there! I have a job that is typically contingent based on project budgets, so over time, I have accumulated quite the collection of scoop installations. I am hoping someone here is knowlegeable enough to help me merge the scoop installations I have (I understand it  may be a rather manual and/or incremental process, and I am okay with that), and then make the merged one as portable as possible. I've been hunting for solutions for over a year with little success, so I figured I should probably come to the group that knows this application the best. Thank you in advance for any help you can offer. 😊

**Comment by HUMORCE** [comment]
See https://github.com/ScoopInstaller/Scoop/discussions/5542

Data for some apps may need to be moved manually. (e.g. data in $env:APPDATA)
